### PR TITLE
Add support for packet size in tcp connectivity tests

### PR DESCRIPTION
**Pull request by yboaron**
Allow setting packet size in e2e tcp connectivity tests, this might help with troubleshooting MTU issues.

### CODE REVIEW

**Comment by yboaron** [comment]
cc @sridhargaddam

**Comment by yboaron** [comment]
cc @sridhargaddam

File: test/e2e/framework/network_pods.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ type NetworkingType bool
 const (
 	HostNetworking NetworkingType = true
 	PodNetworking  NetworkingType = false
+	dataPrefixSize uint           = 25
```
Comment (skitt) [inline test/e2e/framework/network_pods.go]: What does this correspond to? Specifically, how is the value “25” calculated?

**Review by skitt — CHANGES_REQUESTED** [review]

**Review by skitt — APPROVED** [review]

File: test/e2e/framework/network_pods.go (hunk 1 above)
Comment (yboaron) [reply test/e2e/framework/network_pods.go]: TCP payload [1]  composed of "[dataplane] listener says $uuid.string" or  "[dataplane] connector says $uuid.string"

The size of prefix ('[dataplane] listener says') is 25
Added a comment 

[1]
[dataplane] listener says LLLLLLLL-LLLL-LLLL-LLLL-LLLLLLLLLLLL
[dataplane] listener says LLLLLLLL-LLLL-LLLL-LLLL-LLLLLLLLLLLL
[dataplane] listener says LLLLLLLL-LLLL-LLLL-LLLL-LLLLLLLLLLLL
[dataplane] listener says LLLLLLLL-LLLL-LLLL-LLLL-LLLLLLLLLLLL

**Comment by yboaron** [comment]
cc  @sridhargaddam @tpantelis

**Review by tpantelis — APPROVED** [review]

**Pull request merged**